.swift-version

```diff
@@ -1 +1 @@
-3.0
\ No newline at end of file
+4.2
```

Comment:
since this is deprecated, we could replace it with `s.swift_version = '4.2';` in the Podspec file. @iSkelzor  what do you think?